torch/_inductor/codegen/wrapper.py

```diff
@@ -212,6 +220,12 @@ def plan(self, state: MemoryPlanningState):
 
     def codegen(self, code: IndentedBuffer):
         assert self.node.get_name() not in V.graph.removed_buffers
+        for s in self.node.get_size():
+            s_str = str(s)
```

Comment:
You don't have to stringify; instead, test if you have a Symbol and that Symbol is data dependent, and if so make the binding.